CHANGES.rst

```diff
@@ -135,6 +135,11 @@ New features
 
   Thanks to Markus Bach.
 
+* Added calculation of vector diagram contributions to 
+  :math: `(g-2)/2`.
+
+  Thanks to Douglas Jacob.
+
```

Comment:
This has to be move. @Expander, is this supposed to be a part of 2.7.0 release (together with mW tweaks) or a later one?

Reply:
Ah, yes! It has to move upwards to the new 2.7.0 release.